tools/ExposureTimesForConstantDepth.ipynb

```diff
@@ -0,0 +1,422 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "One idea put forward is to vary exposure time in order to keep constant depth. What range of exposure times would be needed?"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Check range of delivered m5 values.\n",
+    "import lsst.sims.maf.db as db"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "opsdb = db.OpsimDatabase('db/kraken_2026.db')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "u 21.6014854524 24.0837842178 2.4822987654\n",
+      "g 22.8128084211 25.4036678373 2.59085941619\n",
+      "r 22.4302863843 24.9637526347 2.53346625048\n",
+      "i 21.9573319547 24.4080723472 2.45074039248\n",
+      "z 20.8779608031 23.7375564311 2.85959562795\n",
+      "y 20.0817199477 22.735065822 2.65334587434\n"
+     ]
+    }
+   ],
+   "source": [
+    "for f in ('u', 'g', 'r', 'i', 'z', 'y'):\n",
+    "    simdata = opsdb.fetchMetricData(['fiveSigmaDepth'], sqlconstraint='filter=\"%s\"' % f)\n",
```

Comment:
@egawiser suggested focusing on the WFD visits here, I think

Reply:
You could issue an enhancement here, instead of making this change now, potentially.